llvmDebugInfoC/src/scripts/konan_lldb.py

```diff
@@ -23,34 +23,48 @@
 
 import lldb
 import struct
+import os
+
+def is_appcode():
+    return 'AppCode' in os.getenv('XPC_SERVICE_NAME', '')
+
+def is_xcode():
+    return 'Xcode' in os.getenv('CLIENT_TYPE', '')
 
 NULL = 'null'
 
+CLIENT_APPCODE = is_appcode()
+CLIENT_XCODE = is_xcode()
+VISUAL_DEBUGGER = CLIENT_APPCODE or CLIENT_XCODE
+
 def log(msg):
     if False:
         print(msg)
 
+def exelog(stmt):
```

Comment:
Separate log for executed statements. Cutting down on these is the primary optimization technique, and there may be many, so put in a file is easier to manage.